controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -84,15 +85,6 @@
 import io.pravega.controller.util.Config;
 import io.pravega.controller.util.RetryHelper;
 import io.pravega.shared.NameUtils;
-import io.pravega.shared.controller.event.ControllerEvent;
```

Comment:
Please revere these changes. This is a checkstyle bug

Reply:
all imports have been organized now.